### PR TITLE
12 fix shapes

### PR DESCRIPTION
**Pull request by piensaenpixel**
Could you take a look @javierarce

### CODE REVIEW

File: src/scss/cdb-components/shapes/error-circle.scss [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      <div class="CDB-Shape-close is-small is-white"></div>
+    </div>
+  </div>
+</div>
+```
+*/
+
+@import '../../cdb-variables/colors';
+
+.CDB-Shape-CircleItem {
```
Comment (javierarce) [inline src/scss/cdb-components/shapes/error-circle.scss:60]: The `x` looks a bit misaligned in Chrome (48.0.2564.97) and Firefox (42.0)

screen shot 2016-02-01 at 11 16 41

File: src/scss/cdb-components/shapes/arrow.scss [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+</div>
+<div style="padding: 20px; background: #2E3C43">
+  <div class="CDB-Shape-Arrow is-white"></div>
+</div>
+```
+*/
+
+@import '../../cdb-variables/colors';
+
+.CDB-Shape-Arrow {
```
Comment (javierarce) [inline src/scss/cdb-components/shapes/arrow.scss:21]: I think we should make the direction that the arrow is pointing explicit with a class (is-up, is-down, etc.), like the toggle arrows.

File: src/scss/cdb-components/shapes/subtract.scss [hunk 3]
```diff
@@ -0,0 +1,85 @@
+/* SG
+# Shapes/Subtract
```
Comment (javierarce) [inline src/scss/cdb-components/shapes/subtract.scss]: I propose to change the name (if possible) to `dash`.

Comment (piensaenpixel) [inline src/scss/cdb-components/shapes/subtract.scss]: +1

File: src/scss/cdb-components/shapes/arrow.scss (hunk 2 above)
Comment (piensaenpixel) [inline src/scss/cdb-components/shapes/arrow.scss:21]: ok

File: src/scss/cdb-components/shapes/error-circle.scss (hunk 1 above)
Comment (piensaenpixel) [inline src/scss/cdb-components/shapes/error-circle.scss:60]: image

**Comment by javierarce** [comment]
:+1: :flags:

**Pull request merged**